api/pyproject.toml

```diff
@@ -1,55 +1,61 @@
-[tool.poetry]
+[project]
 name = "mysagw"
-version = "0.5.1"
 description = "Application management for SAGW"
-repository = "https://github.com/adfinis/mySAGW"
-authors = ["Adfinis"]
 license = "GPL-3.0-or-later"
+version = "0.5.1"
+authors = [{name = "Adfinis"}]
+requires-python = '>=3.13,<4.0'
+dependencies = [
+    "django (>=5.2.7,<6.0.0)",
```

Comment:
Why specify version ranges here? Isn't it nicer to have fixed versions for an app?